### PR TITLE
Add possibility to overwrite connect_timeout argument

### PR DESCRIPTION
**Pull request by miloszbednarzak**
## Problem

30 seconds MySQL's parameter `connect_timeout` in my use cases was not enough and sometimes resulted in the error `Lost connection to MySQL server`. 

## Proposed changes

Proposed changes give the possibility to overwrite this parameter and to set this value according to one's needs.

## Types of changes

What types of changes does your code introduce to PipelineWise?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] Description above provides context of the change
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Unit tests for changes (not needed for documentation changes)
- [ ] CI checks pass with my changes
- [ ] Bumping version in `setup.py` is an individual PR and not mixed with feature or bugfix PRs
- [ ] Commit message/PR title starts with `[AP-NNNN]` (if applicable. AP-NNNN = JIRA ID)
- [ ] Branch name starts with `AP-NNN` (if applicable. AP-NNN = JIRA ID)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions

### CODE REVIEW

File: tap_mysql/connection.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def __init__(self, config):
             "host": config["host"],
             "port": int(config["port"]),
             "cursorclass": config.get("cursorclass") or pymysql.cursors.SSCursor,
-            "connect_timeout": CONNECT_TIMEOUT_SECONDS,
+            "connect_timeout": config.get("connect_timeout", DEFAULT_CONNECT_TIMEOUT_SECONDS) ,
```
Comment (Samira-El) [inline tap_mysql/connection.py:92]: can you cast this to int? just in case `connect_timeout' is passed as stringified integer

**Review by Samira-El — CHANGES_REQUESTED** [review]
Hi, thanks for the PR, I see no problem with the idea of making this configurable.

Can you add add some unit tests to cover this change?
Also, if you could add this to one of the integration tests that would be great.

File: tap_mysql/connection.py (hunk 1 above)
Comment (miloszbednarzak) [reply tap_mysql/connection.py]: sure

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+Unreleased
+------------------
+* Add possibility to overwrite `connect_timeout` argument
+
```
[the remark this answers is not in the document]
Comment (miloszbednarzak) [reply CHANGELOG.md]: done

**Pull request closed without merging**